src/configs/docker_context.rs

```diff
@@ -0,0 +1,25 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct DockerContextConfig<'a> {
+    pub symbol: SegmentConfig<'a>,
+    pub context: SegmentConfig<'a>,
+    pub style: Style,
+    pub only_with_files: bool,
+    pub disabled: bool,
+}
+
+impl<'a> RootModuleConfig<'a> for DockerContextConfig<'a> {
+    fn new() -> Self {
+        DockerContextConfig {
+            symbol: SegmentConfig::new("🐳 "),
+            context: SegmentConfig::default(),
+            style: Color::Blue.bold(),
+            only_with_files: false,
+            disabled: true,
```

Comment:
What do you think about switching these so the module is enabled by default but only when there is are some docker related files?

```suggestion
            only_with_files: true,
            disabled: false,
```